### ISSUE TITLE
Ignore unused function arguments in abstract methods

### ISSUE THREAD

**Issue by sirbrillig** [sirbrillig/phpcs-variable-analysis#90]
@davidrothstein suggested this, and I think it might be a good idea. When an abstract class has methods, those methods may have arguments but no function body (essentially a function signature). We may want to ignore those unused variable warnings.

Example:

```php
<?php

abstract class MasterTestCase {
  protected function doSomething( int $foo, array $bar ) { // this will report two warnings
  }
}
```

**Comment by sirbrillig** [comment]
Duplicate of https://github.com/sirbrillig/phpcs-variable-analysis/issues/133. Closing this one in favor of that.
